Formula/slither-analyzer.rb

```diff
@@ -148,18 +153,13 @@ class SlitherAnalyzer < Formula
   end
 
   resource "protobuf" do
-    url "https://files.pythonhosted.org/packages/e5/9d/20e9bf4067e85c3074f1f5bac820a3cfb9ce885cddd8a649fe3570659c77/protobuf-4.23.3.tar.gz"
-    sha256 "7a92beb30600332a52cdadbedb40d33fd7c8a0d7f549c440347bc606fb3fe34b"
-  end
-
-  resource "pypandoc" do
```

Comment:
The build failure can likely be resolved by re-adding this pypandoc version

Reply:
For context, here's the upstream [issue](https://github.com/ethereum/pyrlp/issues/137) and [PR](https://github.com/ethereum/pyrlp/pull/136) on pyrlp.

Reply:
make sense.

Reply:
I see you readded it but with a newer version; pypandoc's `convert` (the missing function causing the build failure) was removed in 1.8, that's why the version pinned in the formula is 1.7.5. Can you roll it back to the previous version? thanks!

Reply:
I think upstream should also pin to use 1.7.5 as well?

Reply:
Yeah probably! But ultimately the real fix would be for `pyrlp` to discontinue using `setuptools-markdown` which has been deprecated in favor of the builtin `setuptools` functionality. (the dependency goes something like slither-analyzer -> ... -> pyrlp -> setuptools_markdown (deprecated) -> pypandoc)